processor/tailsamplingprocessor/README.md

```diff
@@ -15,6 +15,7 @@ Multiple policies exist today and it is straight forward to add more. These incl
 - `always_sample`: Sample all traces
 - `latency`: Sample based on the duration of the trace. The duration is determined by looking at the earliest start time and latest end time, without taking into consideration what happened in between.
 - `numeric_attribute`: Sample based on number attributes
+- `percentage`: Sample a percentage of traces. Only traces that have not been sampled yet by another policy are taken into account.
```

Comment:
Why would I use this instead of the probabilistic sampling processor? 
https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/probabilisticsamplerprocessor

Reply:
Oh, wasn't aware of the probabilistic sampling processor (for some reason) 🤦🏻  I'll take a look at it and make sure our terminology is consistent.

I think the main difference is when the sampling decision happens. If you only want percentage or probabilistic sampling it doesn't make sense to use this tail sampling processor.
But combining this with other tail sampling policies makes it more meaningful. For instance:

```yaml
    policies:
      [
          {
            name: all-errors,
            type: status_code,
            status_code: {status_codes: [ERROR]}
          },
          {
            name: half-of-remaining,
            type: percentage,
            percentage: {percentage: 0.5}
          },
     ]
```

This pipeline would sample all traces with status code error and 50% of the remaining traces. Using the probabilistic sampling processor you risk dropping traces with errors.

Reply:
Good point. Make sure to mention that in the readme then.

Reply:
Is there a difference between this and just putting the probabilistic sampling processor after tail sampling at 50%?

Reply:
The result will be slightly different: putting the probabilistic sampling processor after the tail sampling processor will filter what the tail sampling processor samples.
The percentage sampler as implemented here will filter what is not sampled by another policy.

So for instance a pipeline:
```
tail sampling (sample all errors) -> probabilistic (at 50%)
```

Result: 50% of traces with error
Why:
- the tail sampler drops every non-error
- the probabilistic sampler drops 50% of what the tail sampler returns

While the following:
```
tail sampling (sampler all errors -> sample 50%)
```

Result: all traces with error + 50% of traces without